tests/acceptance/test_dcos_zookeeper.py

```diff
@@ -24,3 +24,8 @@ def test_delete_zk_node():
     uninstall_package_and_wait('marathon')
 
     assert delete_zk_node('universe/marathon-user')
+
+
+def get_zk_node_data():
+    install_package_and_wait('marathon')
+    get_zk_node_data('universe/marathon-user')
```

Comment:
this isn't a proper test:
1. it needs to have a `test_` prefix for the test to be included
2. it needs to assert something.. in this case asserting the expected value.